README.md

```diff
@@ -88,6 +88,15 @@ class IdentityEntity implements IdentityEntityInterface
 }
 ```
 
+### 3.)  Update authorization view
```

Comment:
This is required only when we want to use nonce, maybe it can be stated in the doc?  (Reading like that I think all user would think they have to set the nonce params)

```suggestion
### 3.)  Update authorization view when using "nonce"
```

Reply:
I see. I've adjusted the readme. What do you think?